test/run_as_user/run_as_user_test.go

```diff
@@ -0,0 +1,64 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build linux && integration
+// +build linux,integration
+
+package run_as_user
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/aws/amazon-cloudwatch-agent-test/test"
+)
+
+const (
+	// Let the agent run for 15 seconds. This will give agent enough time to change user
+	agentRuntime     = 15 * time.Second
+	configOutputPath = "/opt/aws/amazon-cloudwatch-agent/bin/config.json"
+	pidFile          = "/opt/aws/amazon-cloudwatch-agent/var/amazon-cloudwatch-agent.pid"
+	root             = "root"
+	cwagent          = "cwagent"
+)
+
+type input struct {
+	user      string
+	dataInput string
+}
+
+func TestBundle(t *testing.T) {
+
+	parameters := []input{
+		{dataInput: "resources/default.json", user: root},
+		{dataInput: "resources/root.json", user: root},
+		{dataInput: "resources/cwagent.json", user: cwagent},
```

Comment:
IMO, it would be better if we can validate with `unit test` and reduces the amount of `integration test`? (e.g if we has unit test that validates `root` and `cwagent` that is being set in the toml config or json config, does it matter if we confirm root as default and `root` anymore?

Reply:
There are already unit tests for config.

Reply:
Yeah so that's why should we test for `root` anymore?

Reply:
I'm not sure I understand the question. Are you saying we should not test the default? Are you saying we should not test setting the user to root?

Reply:
Yes. Test as "user" cwagent is enough for `integration test` since I dont see any value in testing 'root' when user cwagent runs successfuly

Reply:
I spoke with the team and they think it is better to leave in the root tests.